### PR TITLE
RAS-1110: Update dev dependencies in ROPs packages and get make minify work

### PR DESCRIPTION
**Pull request by SteveScorfield**
# What and why?
Make minify was not working with its current dependencies. This PR fixes this issue.

# How to test?
In the README.me run the "make minify-install make minify" command (just press the green play button). 

# Jira
https://jira.ons.gov.uk/browse/RAS-1110

### CODE REVIEW

**Review by anwilkie — DISMISSED** [review]
The minify command still fails when I run it: `TypeError: ["webpack.config",".webpack/webpack.config",".webpack/webpackfile"].flatMap is not a function`. Not sure if I've missed something else I was supposed to do beforehand. I did the command both before and after deleting the dev dependencies and the package-lock.json file, as described in the README.

**Comment by SteveScorfield** [comment]
> The minify command still fails when I run it: `TypeError: ["webpack.config",".webpack/webpack.config",".webpack/webpackfile"].flatMap is not a function`. Not sure if I've missed something else I was supposed to do beforehand. I did the command both before and after deleting the dev dependencies and the package-lock.json file, as described in the README.

@anwilkie you may need to run npm install first

**Comment by anwilkie** [comment]
> > The minify command still fails when I run it: `TypeError: ["webpack.config",".webpack/webpack.config",".webpack/webpackfile"].flatMap is not a function`. Not sure if I've missed something else I was supposed to do beforehand. I did the command both before and after deleting the dev dependencies and the package-lock.json file, as described in the README.
> 
> @anwilkie you may need to run "npm install" first

This command doesn't work, either, even though I've updated and reinstalled node, set up a fresh pipenv install, and run `nvm use`. I'm going to dismiss my own review, since this might be an issue on my end rather than with the PR.

File: response_operations_ui/static/css/main.css [hunk 1]
```diff
@@ -1 +1 @@
...
...margin:0 0.5rem;padding:0.25rem 1rem;width:auto}.page__content{overflow-x:visible}.ru-survey{margin:1rem 0;border:0.5rem solid #eeecec}.ru-survey h3{font-weight:500}.response__changed{background-color:#edf4f0}.ce-section h2{padding-top:9px;margin-bottom:0.3rem;padding-bottom:0.4rem}.ce-events__list{width:100%;border-collapse:collapse;margin:1.3rem 0 0.3rem auto}.ce-events__list th,.ce-events__list td{text-align:left;vertical-align:top;padding:0.5rem 0;border-bottom:1px solid #ccc}.ce-events__list th{font-weight:700;border-bottom:1px solid #222}#upload legend{display:none}#filedrag{display:none;font-weight:bold;text-align:center;padding:1em 0;margin:1em 0;color:#555;border:2px dashed #555;border-radius:7px;cursor:default}#filedrag.hover{color:#0f8243;border-color:#0f8243;border-style:solid;box-shadow:inset 0 3px 4px #888}.ce-overview-table{table-layout:fixed}.ce-overview-table th td{width:33.3%}.error-message{color:#d0021b}.ons-tabs__list{width:100%}.ons-tabs__list--row::after{background:var(--ons-color-borders);bottom:0;-webkit-box-shadow:0 1px 0 0 var(--ons-color-page-light);box-shadow:0 1px 0 0 var(--ons-color-page-light);content:"";height:1px;left:0;position:absolute;width:100%}
```
[the remark this answers is not in the document]
Comment (SteveScorfield) [reply response_operations_ui/static/css/main.css:1]: So it appears that some CSS variable values have changed from HTML colour codes to a human readable value e.g #000000 has been changed to black. It has also added missing quotations to string names e.g. role=button is now role="button"

File: Makefile [hunk 2]
```diff
@@ -46,4 +46,4 @@ minify-install:
 	npm install -D babel-loader @babel/core @babel/preset-env webpack
 
 minify:
-	npx webpack
+	npx webpack --mode=production
```
Comment (SteveScorfield) [inline Makefile:49]: This removes the "The 'mode' option has not been set...." warning.

File: README.md [hunk 3]
```diff
@@ -97,7 +97,7 @@ It is a manual step to minimise the JavaScript
 make minify-install
 make minify
 ```
-
+To recreate all dependencies, delete everything between ""devDependencies": {}" in the package.json, delete the package-lock.json and run the above commands.
```
[the remark this answers is not in the document]
Comment (SteveScorfield) [reply README.md]: Removed

File: package-lock.json [hunk 4]
```diff
@@ -9,139 +9,173 @@
       "version": "1.0.0",
       "license": "ISC",
       "dependencies": {
-        "@cypress/request": "^3.0.0",
         "abbrev": "^1.1.1",
-        "acorn": "^8.10.0",
+        "acorn": "^8.11.3",
```
[the remark this answers is not in the document]
Comment (SteveScorfield) [reply package-lock.json]: Reverted

**Review by arroyoAle — APPROVED** [review]
Looks fine to me, commands run fine on Macbook but cause issues in iRex (seems like it doesn't like my version of node yet, so I hope I don't need to use this on iRex until they add support)

**Pull request merged**